docker-compose.yml

```diff
@@ -49,13 +49,13 @@ services:
      - template_volume:/usr/share/nginx/html/templates
      - guidance_volume:/usr/share/nginx/html/guidance
   admin:
-    image: ${REGISTRY}/montagu-admin-portal:master
+    image: ${ORG}/montagu-admin-portal:master
     networks:
      - proxy
     depends_on:
      - api
   static:
-    image: ${REGISTRY}/montagu-static:master
+    image: docker.montagu.dide.ic.ac.uk:5000/montagu-static:master
```

Comment:
Note open ticket for moving or deprecating montagu-static https://mrc-ide.myjetbrains.com/youtrack/issue/VIMC-4230